sdk/ml/azure-ai-ml/azure/ai/ml/_artifacts/_blob_storage_helper.py

```diff
@@ -221,6 +224,13 @@ def download(
                     continue
 
                 blob_content = self.container_client.download_blob(item)
+
+                # check if total size of download has exceeded 100 MB
+                full_storage_url = f"https://{self.account_name}.blob.core.windows.net/{self.container}/{starts_with}"
```

Comment:
Same issue of hard coded blob endpoint for public cloud.

Reply:
Addressed offline and fixed in code.